### PR TITLE
[ISSUE #4101] Fix Javadoc for ClientBuilder.keepAliveInterval

### LINKED ISSUE #4101: Fix Javadoc for ClientBuilder.keepAliveInterval
**Describe the bug**
During the change in https://github.com/apache/pulsar/pull/1547, there was some copy and pasting that overlooked inconsistencies in variable naming and javadocs.
```
    /**
     * Set keep alive interval in seconds for each client-broker-connection. <i>(default: 30)</i>.
     *
     * @param keepAliveIntervalSeconds
     * @param unit time unit for {@code statsInterval}
     * @return the client builder instance
     */
    ClientBuilder keepAliveInterval(int keepAliveIntervalSeconds, TimeUnit unit);

```
`keepAliveIntervalSeconds` should now be `keepAliveInterval` in the javadoc and variable name until it's converted into seconds
`@param unit time unit for {@code statsInterval}` should be `@param unit time unit for {@code keepAliveInterval}`

### PR DESCRIPTION
**Pull request by ambition119**
Fixes #4101 

### Motivation

fix issue

### Modifications

fix keepAliveInterval method java doc

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java [hunk 1]
```diff
@@ -337,7 +337,7 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * Set keep alive interval in seconds for each client-broker-connection. <i>(default: 30)</i>.
      *
      * @param keepAliveIntervalSeconds
-     * @param unit time unit for {@code statsInterval}
+     * @param unit the time unit in which the keepAliveIntervalSeconds is defined
      * @return the client builder instance
      */
     ClientBuilder keepAliveInterval(int keepAliveIntervalSeconds, TimeUnit unit);
```
Comment (sijie) [inline pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java]: we need remove seconds from `keepAliveIntervalSeconds` and also update the javadoc.

Comment (ambition119) [reply pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java]: ok

File: pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java [hunk 2]
```diff
@@ -294,7 +294,7 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * Set the interval between each stat info <i>(default: 60 seconds)</i> Stats will be activated with positive
      * statsIntervalSeconds It should be set to at least 1 second
```
Comment (sijie) [inline pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java]: change `statsIntervalSeconds` to `statsInterval`

File: pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java [hunk 3]
```diff
@@ -336,11 +336,11 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
     /**
      * Set keep alive interval in seconds for each client-broker-connection. <i>(default: 30)</i>.
```
Comment (sijie) [inline pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java]: `(default: 30)` to `(default: 30s)`?

Comment (sijie) [inline pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java]: remove "in seconds"?

**Comment by ambition119** [comment]
run cpp test

**Comment by ambition119** [comment]
run cpp Tests

**Pull request merged**